test/test_licensevalidator.py

```diff
@@ -101,56 +98,56 @@ def test_javascript():
     assert origin_vs_deps["JavaScript"][1].version == "4.6.3"
 
 
-def test_cpp():
-    """Test cpp project."""
-    result, origin_vs_deps = validate_used_licenses(
-        "./testbench/multilang/cpp-proj",
-        [{"path": ".", "cpp-conan-included-profile-files": ["./conan_profile"]}],
-        "../whitelist.txt",
-    )
+# def test_cpp():
```

Comment:
Shall we just delete this?

Reply:
Currently there is a bug in the pivotal checker for cpp and thats why I just commented out the tests instead of removing them